nncf/experimental/tensor/functions.py

```diff
@@ -48,7 +48,7 @@ def device(a: TTensor) -> TensorDeviceType:
 
 @functools.singledispatch
 @_tensor_guard
-def squeeze(a: TTensor, axis: Optional[Union[int, Tuple[int]]] = None) -> TTensor:
+def squeeze(a: TTensor, axis: Optional[Union[int, Tuple[int]]] = None) -> Tensor:
```

Comment:
Functions in this file always return Tensor or list of Tensor

Reply:
I think this is not entirely correct. I mean these functions return a `Tensor` object only if there is no registered version for the type of the passed argument.

Reply:
Update all return type to `Tensor` to correctly works "pop-up suggestions" (or how it names when editor suggest functions by first symbols?).
So set Tensor type to first argument. But not sure about second, added like Union[torch.Tensor, float], because it can be used like `torch.tensor(1) + 1` or `fns.min(torch.tensor(1), 0)`. May be do you have anny suugestion about it?